playbooks/cluster/openshift/config.yml

```diff
@@ -82,7 +82,7 @@
 
 - import_playbook: "{{ openshift_ansible_dir }}/playbooks/prerequisites.yml"
 
-- hosts: all
+- hosts: nodes
```

Comment:
Should be nodes and masters (they are part of the `OSEv3` group)

Reply:
all masters are nodes

Reply:
I can still change them to 'masters nodes' to be more clear if you like though

Reply:
> I can still change them to 'masters nodes' to be more clear if you like though

it will be a good solution